blog/_posts/2021-08-20-symbiosis-symbulation.md

```diff
@@ -0,0 +1,65 @@
+# Symbiosis in Symbulation
+#### August 5, 2021 Tiffany-Ellen Vo
+- - - 
+
+[comment]: <> (How do I make it so the text is aligned with the picture?)
+
+><img src="https://scontent-sjc3-1.xx.fbcdn.net/v/t1.6435-9/106543962_3538908449476892_6632988677477612051_n.jpg?_nc_cat=110&ccb=1-4&_nc_sid=09cbfe&_nc_ohc=MVm9XlfPXKYAX8k78kl&_nc_ht=scontent-sjc3-1.xx&oh=aff9f72aa230c94ddce73a0e5ab28c3e&oe=6131225E" alt="drawing" width="200"/> Hello! My name is Tiffany-Ellen Vo, I'm currently a senior at UC Santa Cruz, studying computer engineering. I’m especially interested in logic design, robotics, and computer architecture. In my free time, I love rock climbing, baking, and crocheting. To see some of my other projects, check out my GitHub!
+
+- - - 
+
+## Introduction to Symbiosis 
+ Through the guidance of my mentor Anya, this summer I have worked on simulating symbiosis, the interaction between two organisms that are often referred as the host and its symbiont. While symbiosis has been simulated using Symbulation, the purpose of this project is to simulate symbiosis using the Empirical library in order to create and understand various types of symbiosis relationships while specificallly using the Avida complex genome to make the host and its symbionts. For this summer, we focused on being able to add symbionts to a host system which would then increase the count of a point system. The original idea is that if the host reaches a certain amount of points, it would then donate a point to the symbiont and if the symbiont reaches a certain amount of points, then the symbiont would donate a point to the host which would then be multiplied by three.
+
+---
+
+ ## Symbiosis Environment
+ In order to create symbiosis, we need to create a host with symbionts and create a world to put them in. Header files for the world, host and symbiont were created in order to do so. These files were originally created by the last WAVES participant that worked with Anya, which were then built upon and edited for this summer. 
```

Comment:
Be sure to mention the science/functionality objectives this work is enabling

Reply:
Agreed, and this seems best to put in the earlier section. The important point would be that Symbulation's original Host/Symbiont are very simple and this project's goal is to figure out how to incorporate Avida's more complex genomes to enable more varied and interesting forms of symbiosis